### PR TITLE
signal-desktop: 6.10.1 -> 6.11.0, signal-desktop-beta: 6.11.0-beta.2 -> 6.12.0-beta.1

### PR DESCRIPTION
**Pull request by equirosa**
###### Description of changes

New releases:

- signal-desktop: 
  - https://github.com/signalapp/Signal-Desktop/releases/tag/v6.11.0
  - https://github.com/signalapp/Signal-Desktop/releases/tag/v6.12.0
- signal-desktop-beta:
  - https://github.com/signalapp/Signal-Desktop/releases/tag/v6.12.0-beta.1
  - https://github.com/signalapp/Signal-Desktop/releases/tag/v6.13.0-beta.1

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by equirosa** [comment]
Important question: do we want to make these into two separate commits? I know the norm is to make it a single commit; but, since these are released at almost the same time consistently, I'm not sure if we have any preference regarding this.

**Comment by equirosa** [comment]
New versions were released a little while ago, I'll get them in ASAP. Though it seems the mirrors aren't up yet so I can't build the packages until they are.

**Comment by equirosa** [comment]
converting to draft, since signal-desktop-beta still isn't available from the mirrors

**Comment by equirosa** [comment]
ok, _now_ it's ready :)

**Review by Zahrun — CHANGES_REQUESTED** [review]
```
{ callPackage }: builtins.mapAttrs (pname: attrs: callPackage ./generic.nix (attrs // { inherit pname; })) {
  signal-desktop = {
    dir = "Signal";
    version = "6.13.0";
    hash = "sha256-YnWPrbC5p/Y4gTfKjcBNP86wrD031RtEJb3gQt1GqU8=";
  };
  signal-desktop-beta = {
    dir = "Signal Beta";
    version = "6.14.0-beta.1";
    hash = "sha256-17yMRCuWcg/H+HdTg+x24FGryMftE2iYpJCR5Waauwc=";
  };
}
```

File: pkgs/applications/networking/instant-messengers/signal-desktop/default.nix [hunk 1]
```diff
@@ -1,12 +1,12 @@
 { callPackage }: builtins.mapAttrs (pname: attrs: callPackage ./generic.nix (attrs // { inherit pname; })) {
   signal-desktop = {
     dir = "Signal";
-    version = "6.10.1";
-    hash = "sha256-uWwRgP9iYirZU9x3QtS5lRGI7vLOOtX4B4fgVuYxkho=";
+    version = "6.12.0";
+    hash = "sha256-nhUrxwcE53QG2akSHCFKJR62ydwZ7d4LI+l4o1bgKJo=";
```
Comment (Zahrun) [inline pkgs/applications/networking/instant-messengers/signal-desktop/default.nix:5]:
```suggestion
    version = "6.13.0";
    hash = "sha256-YnWPrbC5p/Y4gTfKjcBNP86wrD031RtEJb3gQt1GqU8="";
```

File: pkgs/applications/networking/instant-messengers/signal-desktop/default.nix [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
   };
   signal-desktop-beta = {
     dir = "Signal Beta";
-    version = "6.11.0-beta.2";
-    hash = "sha256-tw8VsPC0shKIN13ICD0PVKhKxA7rdj16r2lP2UEJGsY=";
+    version = "6.13.0-beta.1";
+    hash = "sha256-An76VcyuDvTVgXL8sVx0El5EognReiiUBTbRxNmMOrQ=";
```
Comment (Zahrun) [inline pkgs/applications/networking/instant-messengers/signal-desktop/default.nix:10]:
```suggestion
    version = "6.14.0-beta.1";
    hash = "sha256-17yMRCuWcg/H+HdTg+x24FGryMftE2iYpJCR5Waauwc="";
```

**Comment by Zahrun** [comment]
6.14 and 6.15 beta are out

**Comment by equirosa** [comment]
Closing it since a different PR was merged :)

**Pull request closed without merging**